helpers/favorite_format.js

```diff
@@ -0,0 +1,15 @@
+var fetch = require('node-fetch');
+var pry = require('pryjs');
+var Forecast = require('./forecast')
+
+
+module.exports = class FavoriteFormat extends Forecast{
```

Comment:
As Cory said the other day, is inheritance a good idea or a bad idea here? I don't have an opinion, just wondered what your thought process was :)